source/aws_profile.c

```diff
@@ -620,7 +641,8 @@ static int s_profile_collection_add_profile(
     const struct aws_byte_cursor *profile_name,
     bool has_prefix,
     const struct profile_file_parse_context *context,
-    struct aws_profile **current_profile_out) {
+    struct aws_profile **current_profile_out,
+    const enum aws_profile_section_type section_type) {
```

Comment:
similar to earlier comment, move `section_type` arg so that it comes before `profile_name`

Reply:
also, debatable, but does it make to code more clear going forward if we change ~profile_name~ -> `section_name` or `name`?

Reply:
I think it would be better to stick to one name instead of mixing and matching.